cmd/application/update/update.go

```diff
@@ -56,6 +58,10 @@ const (
 	flagBuiltIn      = "built-in-catalog"
 	flagBuiltInShort = "b"
 	flagBuiltInHelp  = "Update the built-in catalog in the ocne-system namespace."
+
+	flagCatalogName      = "catalog"
```

Comment:
Please update the manpage for update https://github.com/oracle-cne/ocne/blob/main/doc/manpages/ocne-application.md?plain=1#L93

Reply:
Good catch, thank you.  Fixed.